### PR TITLE
CORE-316 Update instantlaunches endpoints for app versions

### PR DESCRIPTION
**Pull request by psarando**
This PR will update the queries used by the `/instantlaunches/*full` and ` /instantlaunches/quicklaunches/public` endpoints for the app versions migrations in cyverse-de/de-database#10.

It will also add the `app_version_id` to Instant Launch endpoint responses.

### CODE REVIEW

File: instantlaunches/db.go [hunk 1]
```diff
@@ -444,20 +450,19 @@ const listPublicQLsQuery = `
...
 		ql.name,
 		ql.description,
 		ql.is_public,
 		s.submission
 	FROM quick_launches ql
-	JOIN app_listing a on ql.app_id = a.id
 	JOIN users u on ql.creator = u.id
 	JOIN submissions s on ql.submission_id = s.id
-	WHERE u.username = $1 OR ( ql.is_public = true AND a.is_public = true)
+	WHERE u.username = $1 OR ql.is_public = true
```
Comment (psarando) [inline instantlaunches/db.go:461]: The `app_listing.is_public` flag is obsolete and will be dropped by cyverse-de/de-database#10.

**Review by ianmcorvidae — APPROVED** [review]
Looks reasonable to me.

**Review by slr71 — APPROVED** [review]
Looks good to me! :shipit:

**Review by johnworth — APPROVED** [review]
Seems good to me. :shipit:

**Comment by psarando** [comment]
Thanks for the reviews! 👍

**Pull request merged**